posthog/models/session_recording_event.py

```diff
@@ -19,3 +19,14 @@ class Meta:
     distinct_id: models.CharField = models.CharField(max_length=200)
     session_id: models.CharField = models.CharField(max_length=200)
     snapshot_data: JSONField = JSONField(default=dict)
+
+
+class SessionRecordingViewed(models.Model):
```

Comment:
I would inherit this from `UUIDModel` to use Michael's UUIDT model and avoid tech debt with the primary keys thing.

Reply:
Not sure what the benefit is in this case - we're doing lookups via an index anyways.